### PR TITLE
Some more compat and convenience for lsusb.py

### PR DESCRIPTION
**Pull request by garloff**
This set of commits provides a few improvements for lsusb.py:
* Search multiple common locations for usb.ids
* Improve parser readability (enums in state machine) and handle faulty 01xy device id (in some distributed versions of usb.ids, sigh)
* Display hidraw and driver names for usbhid devices
* Maintain python2 compatibility (still trivial, fortunately)

### CODE REVIEW

File: lsusb.py.in [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #
 # Copyright (c) 2009 Kurt Garloff <garloff@suse.de>
 # Copyright (c) 2013 Kurt Garloff <kurt@garloff.de>
+# Copyright (c) 2018 Kurt Garloff <kurt@garloff.de>
```
Comment (gregkh) [inline lsusb.py.in]: This should read:
# Copyright (c) 2013,2018 Kurt Garloff <kurt@garloff.de>
no need to have multiple lines.

There's a whole "how to write a valid copyright line" free online tutorial on the linuxfoundation site somewhere, you should take a look at it if you are curious :)

Comment (gregkh) [reply lsusb.py.in]: ok, markdown turned that # character into a huge title line, sorry about that :)

**Comment by garloff** [comment]
OK, it was not so much voodoo after all. I did a rebase where the copyright notice is right with the first commit and where I have the DCO lines in.
Hopefully this meets the criteria for merging now. If not, hints are welcome.

**Comment by gregkh** [comment]
Thanks for the changes, looks great!

**Comment by garloff** [comment]
OK, I commited another change to my fork that changes the shebang back with an appropriate comment.
Greg -- I'm happy to create another PR as soon as we find alignment on this.
-- Kurt

**Pull request merged**